setup.py

```diff
@@ -238,6 +238,7 @@ def _setup_entry_points() -> Dict:
             "deepsparse.instance_segmentation.annotate=deepsparse.yolact.annotate:main",
             f"deepsparse.image_classification.eval={ic_eval}",
             "deepsparse.license=deepsparse.license:main",
+            "deepsparse.validate_license=deepsparse.license:validate_license",
```

Comment:
thoughts on make validate a subcommand of deepsparse license? `deepsparse.license validate`?

Reply:
thought about that, but I think they are different enough user experiences (setting up a license vs making sure the setup is ok), also since the validate cli matches the validate_license function exactly